### DISCUSSION TITLE
File association change after upgrading applications

### DISCUSSION THREAD

**Discussion by jordyverton** [microsoft/winget-cli#3811] [Configuration]
Hi,

I run a PowerShell script that distributes, removes and upgrades apps.
Recently alot of users have let me know that their file associations change after their applications were updated with the used WinGet script.

In the cases I've looked into - it appears that once a browser updates (so far only Chrome and Edge) it makes that browser the standard PDF reader rather than Acrobat Reader for example.

Has anyone else experienced this issue and found a way to avoid this from happening?
Or any idea(s) as to why this is occuring?

Any help is appreciated.

P.S: The line used to upgrade apps is ".\winget.exe upgrade --id $name --silent"
Variable being the ID.

**Comment by Trenly** [comment]
This is most likely due to the application installers themselves changing the default. 

At it's core, winget simply downloads the installer from a publisher's website, checks that the file downloaded matches the expected hash for security, and then runs them using their default switches for an unattended installation.
